src/Statement/StatementList.php

```diff
@@ -105,6 +106,34 @@ public function addNewStatement( Snak $mainSnak, $qualifiers = null, $references
 		$this->addStatement( $statement );
 	}
 
+	public function removeStatement( Statement $statement ) {
```

Comment:
Ok, so this should rather be a `removeByGuid` method I see now. I will update the PR accordingly. Thanks for the notice Jeroen :-)